sklearn/naive_bayes.py

```diff
@@ -775,12 +775,12 @@ class MultinomialNB(_BaseDiscreteNB):
         Class labels known to the classifier
 
     coef_ : ndarray of shape (n_classes, n_features)
-        Mirrors ``feature_log_prob_`` for interpreting `MultinomialNB`
+        Mirrors ``feature_log_prob_[1]`` for interpreting `MultinomialNB`
```

Comment:
This is only true for the binary case. For multi-class case `coef_` and `feature_log_proba_` are the same:

```python
import numpy as np
from sklearn.naive_bayes import MultinomialNB

rng = np.random.RandomState(1)
X = rng.randint(5, size=(6, 100))
y = np.array([1, 2, 3, 4, 5, 6])

clf = MultinomialNB()
clf.fit(X, y)

print(clf.feature_log_prob_.shape)
# (6, 100)

print(clf.feature_log_prob_.shape)
# (6, 100)
```

```suggestion
        Mirrors ``feature_log_prob_`` with a multiclass target, and 
        ``feature_log_prob_[1]`` with a binary target for interpreting
        `MultinomialNB`
```